src/applications/personalization/profile360/selectors.js

```diff
@@ -34,10 +33,8 @@ export function selectVet360SuccessfulTransactions(state) {
   return state.vet360.transactions.filter(isSuccessfulTransaction);
 }
 
-export function selectVet360ErroredTransactions(state) {
-  return state.vet360.transactions.filter(transaction => {
-    return isFailedTransaction(transaction) || isErroredTransaction(transaction);
-  });
+export function selectVet360FailedTransactions(state) {
+  return state.vet360.transactions.filter(isFailedTransaction);
 }
```

Comment:
This selector is the top-level filter for the transaction error banners. It makes the assumption that any transaction that requires an error alert banner to show must have a status of failed or rejected. I believe this is a safe assumption, but BE may have to weigh-in if not.

cc @rlbaker @hpjaj @emersonthis